src/pack.cc

```diff
@@ -402,7 +402,7 @@ unsigned char *drizzle_pack_auth(drizzle_st *con, unsigned char *ptr,
 
     if (con->options.auth_plugin)
     {
-      snprintf((char *)ptr, DRIZZLE_MAX_SCRAMBLE_SIZE, "%s", con->password);
+      strncpy((char *)ptr, con->password, DRIZZLE_MAX_SCRAMBLE_SIZE);
```

Comment:
Just comment for the future, we might want some error handling here as that is a very short password: https://github.com/sociomantic-tsunami/libdrizzle-redux/blob/597d2be70991be25c4545bf8031deb14612231e1/include/libdrizzle-redux/constants.h#L87

Reply:
Ha! I just thought it was very short :)